cluster/examples/kubernetes/ceph/filesystem.yaml

```diff
@@ -22,6 +22,9 @@ spec:
         # gives a hint (%) to Ceph in terms of expected consumption of the total cluster capacity of a given pool
         # for more info: https://docs.ceph.com/docs/master/rados/operations/placement-groups/#specifying-expected-pool-size
         #targetSizeRatio: .5
+        # Allow setting pool with replica 1, this could lead to data loss without recovery.
+        # Make sure you're *ABSOLUTELY CERTAIN* that is what you want
+        #allowUnsafeSingleReplica: true
```

Comment:
In the examples, how about if we set these instead of commenting out:
- filesystem.yaml: `allowUnsafeSingleReplica: false`
- filesystem-test.yaml: `allowUnsafeSingleReplica: true`

We'll need something similar in all the other examples using pools, including the storageclass-test.yaml under the csi folder.

Reply:
Done.